ray-operator/test/e2erayservice/rayservice_upgrade_test.go

```diff
@@ -33,15 +32,15 @@ func TestOldHeadPodFailDuringUpgrade(t *testing.T) {
 
 	test.T().Logf("Waiting for RayService %s/%s to running", rayService.Namespace, rayService.Name)
 	g.Eventually(RayService(test, rayService.Namespace, rayService.Name), TestTimeoutMedium).
-		Should(WithTransform(RayServiceStatus, Equal(rayv1.Running)))
+		Should(WithTransform(IsRayServiceReady, BeTrue()))
 
 	// Get the latest RayService
 	rayService, err = GetRayService(test, namespace.Name, rayServiceName)
 	g.Expect(err).NotTo(HaveOccurred())
 	g.Expect(rayService).NotTo(BeNil())
 
 	curlPodName := "curl-pod"
-	curlContainerName := "curl-container"
+	curlContainerName := "curl"
```

Comment:
Make the linter happy:
<img width="1163" alt="image" src="https://github.com/user-attachments/assets/02da7982-b11b-4d0e-a0f8-723696947f09" />